dd-sdk-android/src/testDebug/kotlin/com/datadog/android/core/internal/utils/RuntimeUtilsDebugTest.kt

```diff
@@ -38,67 +26,18 @@ import org.mockito.quality.Strictness
 @MockitoSettings(strictness = Strictness.LENIENT)
 internal class RuntimeUtilsDebugTest {
 
-    @BeforeEach
-    fun `set up`(){
-        Datadog.initialized.set(true)
-        LogsFeature.initialized.set(true)
-    }
-
-    @AfterEach
-    fun `tear down`() {
-        Datadog.initialized.set(false)
-        LogsFeature.initialized.set(false)
-        Datadog.isDebug = false
-        InternalLogsFeature.stop()
-    }
-
     // region sdkLogger
 
     @Test
-    fun `M build LogCat sdkLogger W buildSdkLogger() {InternalLogs off}`() {
-        // Given
-        InternalLogsFeature.initialized.set(false)
-
-        // When
-        val logger = buildSdkLogger()
-
-        // Then
-        val handler: LogHandler = logger.handler
-        assertThat(handler).isInstanceOf(CombinedLogHandler::class.java)
-        val handlers = (handler as CombinedLogHandler).handlers.toList()
-        assertThat(handlers.filterIsInstance<LogcatLogHandler>())
-            .hasSize(1)
-            .allMatch { it.serviceName == SDK_LOG_PREFIX }
-        assertThat(handlers.filterIsInstance<DatadogLogHandler>())
-            .hasSize(0)
-        assertThat(handlers.filterIsInstance<NoOpLogHandler>())
-            .hasSize(1)
-    }
-
-    @Test
-    fun `M build combined sdkLogger W buildSdkLogger() {InternalLogs on}`() {
-        // Given
-        InternalLogsFeature.initialized.set(true)
-        InternalLogsFeature.persistenceStrategy = mock()
-        whenever(InternalLogsFeature.persistenceStrategy.getWriter()) doReturn mock()
-
+    fun `M build LogCat sdkLogger W buildSdkLogger()`() {
         // When
         val logger = buildSdkLogger()
 
         // Then
         val handler: LogHandler = logger.handler
-        assertThat(handler).isInstanceOf(CombinedLogHandler::class.java)
-        val handlers = (handler as CombinedLogHandler).handlers.toList()
-        println("Handlers : ${handlers.joinToString()}")
-        assertThat(handlers.filterIsInstance<LogcatLogHandler>())
-            .hasSize(1)
-            .allMatch { it.serviceName == SDK_LOG_PREFIX }
-        assertThat(handlers.filterIsInstance<DatadogLogHandler>())
-            .hasSize(1)
-            .allMatch { it.logGenerator.serviceName == InternalLogsFeature.SERVICE_NAME }
-            .allMatch { it.logGenerator.envTag == "env:prod" }
-            .allMatch { it.logGenerator.loggerName == SDK_LOGGER_NAME }
-            .allMatch { it.logGenerator.userInfoProvider is NoOpUserInfoProvider }
+        assertThat(handler).isInstanceOf(LogcatLogHandler::class.java)
+        assertThat((handler as LogcatLogHandler).serviceName)
```

Comment:
Yes, we need to, because `serviceName` is passed in the constructor which is called in the `RuntimeUtils` with the value defined in the `RuntimeUtils` as well.